src/Codeception/Subscriber/Console.php

```diff
@@ -564,7 +644,7 @@ public function detectWidth()
             } else {
```

Comment:
I think that return type can be set to : int after casting result of getenv('COLUMNS')